encord/utilities/image_utilities.py

```diff
@@ -0,0 +1,6 @@
+def rle_encode() -> str:
```

Comment:
shall we remove these for now?

Reply:
Oh, that was a complete accident, great that you've spotted it, removed!